whitelist/regex.lst

```diff
@@ -1,3 +1,3 @@
-^bor\..*\.golem\.network$
-^geth\..*\.golem\.network$
+^bor[0-9]*\.golem\.network$
```

Comment:
do we need to restrict numbers?
This for example does not allow `geth-dev.golem.network` I think `^geth.*\.golem\.network$` is enough.